.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+before_script:
+  - export APP_IMAGE=keboola-functional-tests-scaffolding
+  - docker -v
+  - docker build -t $APP_IMAGE .
+  - docker run $APP_IMAGE composer ci
```

Comment:
Mohlo by se to v rámci toho buildu zkusit nějak spustit nad tím adresářem `tests/functional`.
Zkusil jsem to ale chybí tomu ta appka:
```
docker-compose run --rm dev php /code/src/functional /code/tests/functional

Test "empty-app"
✗ Execution failed unexpectedly (1)
Could not open input file: /code/src/run.php
```

Reply:
ok, souhlas.